hedera-mirror-monitor/src/test/java/com/hedera/mirror/monitor/publish/transaction/account/AccountDeleteTransactionSupplierTest.java

```diff
@@ -0,0 +1,61 @@
+package com.hedera.mirror.monitor.publish.transaction.account;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.hedera.hashgraph.sdk.AccountDeleteTransaction;
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.mirror.monitor.publish.transaction.AbstractTransactionSupplierTest;
+
+class AccountDeleteTransactionSupplierTest extends AbstractTransactionSupplierTest {
+
+    @Test
+    void createWithMinimumData() {
+        AccountDeleteTransactionSupplier accountDeleteTransactionSupplier = new AccountDeleteTransactionSupplier();
+        accountDeleteTransactionSupplier.setAccountId(ACCOUNT_ID.toString());
+        AccountDeleteTransaction actual = accountDeleteTransactionSupplier.get();
+
+        assertThat(actual)
+
+                .returns(ACCOUNT_ID, AccountDeleteTransaction::getAccountId)
+                .returns(MAX_TRANSACTION_FEE_HBAR, AccountDeleteTransaction::getMaxTransactionFee)
+                .returns(AccountId.fromString("0.0.2"), AccountDeleteTransaction::getTransferAccountId)
+                .satisfies(a -> assertThat(a.getTransactionMemo()).contains("Mirror node deleted test account"));
```

Comment:
For the last chain, you can use extract to make this cleaner.
```suggestion
                .extracting(AccountDeleteTransaction::getTransactionMemo)
                .contains("Mirror node deleted test account");
```

Reply:
I've had some issues with `extracting`, I get build errors on this unless I have `toString()` after extracting, it doesn't recognize `contains` otherwise.

Reply:
I guess it assumes ObjectAssert unless you tell it otherwise:
```java
    .extracting(AccountDeleteTransaction::getTransactionMemo, InstanceOfAssertFactories.STRING)
    .contains("Mirror node deleted test account");
```

Reply:
Did that here and in other places where there is a memo (removed transaction memo checks)